pkg/client/restclient/client.go

```diff
@@ -38,6 +38,18 @@ const (
 	envBackoffDuration = "KUBE_CLIENT_BACKOFF_DURATION"
 )
 
+// RESTClient for Kubernetes API
```

Comment:
// Interface captures the set of operations for generically interacting with Kubernetes REST apis.